packages/web3-providers/src/index.ts

```diff
@@ -12,3 +12,8 @@ export const Rarible = new RaribleAPI()
 export const NFTScan = new NFTScanAPI()
 export const CoinGecko = new CoinGeckoAPI()
 export const RSS3 = new RSS3API()
+
+// Method for provider proxy
+export { getOpenSeaNFTList, getOpenSeaCollectionList } from './opensea'
```

Comment:
Access these methods on the API instance.

Reply:
The `getOpenSeaNFTList, getOpenSeaCollectionList` used by proxy. The proxy will provider a apiKey. So need to create a separate instance in `getOpenSeaNFTList, getOpenSeaCollectionList` to use apiKey.